debian/bullseye/8.2/Dockerfile

```diff
@@ -0,0 +1,202 @@
+# syntax = docker/dockerfile:1.0.2-experimental
+ARG SPRYKER_PHP_VERSION=8.2.5
+
+FROM spryker/newrelic-extension:10.9.0.324-php8.2 as spyker-newrelic-arm
+
+FROM php:${SPRYKER_PHP_VERSION}-fpm-bullseye
+
+ARG TARGETPLATFORM
+ARG COMPOSER_VERSION=2.5.5
+ARG BLACKFIRE_VERSION=1.86.8
+ARG BLACKFIRE_PHP_VERSION=82
+ARG NEWRELIC_VERSION=10.9.0.324
+ARG TIDEWAYS_VERSION=5.5.14
+ARG TIDEWAYS_PHP_VERSION=8.2
+
+ENV srcRoot /data
+
+RUN mkdir -p ${srcRoot}
+
+ARG PHP_RUN_DEPS="\
+    libbz2-1.0 \
+    libfreetype6 \
+    libgmp10 \
+    libicu67 \
+    libjpeg62-turbo \
+    libpng16-16 \
+    libwebp6 \
+    libxml2 \
+    libxpm4 \
+    libxslt1.1 \
+    libzip4"
+
+
+ARG PHP_BUILD_DEPS="\
+    libbz2-dev \
+    libfreetype6-dev \
+    libgmp-dev \
+    libicu-dev \
+    libjpeg62-turbo-dev \
+    libpng-dev \
+    libpq-dev \
+    libwebp-dev \
+    libxml2-dev \
+    libxpm-dev \
+    libzip-dev"
+
+ARG PHP_EXTENSIONS="\
+    bcmath \
+    bz2 \
+    gd \
+    gmp \
+    intl \
+    mysqli \
+    opcache \
+    pcntl \
+    pdo_mysql \
+    pdo_pgsql \
+    pgsql \
+    soap \
+    sockets \
+    zip"
+
+ARG PHP_PECL_EXTENSIONS="\
+    apcu \
+    redis"
+
+ARG ADDITIONAL_PHP_PECL_EXTENSIONS="\
+    pcov \
+    xdebug \
+    xhprof"
+
+ARG CFLAGS="-I/usr/src/php"
+RUN apt update -y \
+    && apt install -y \
+    bash \
+    curl \
+    default-mysql-client \
+    g++ \
+    git \
+    gnupg2 \
+    graphviz \
+    locales-all \
+    make \
+    netcat-openbsd \
+    openssh-client \
+    postgresql-client \
+    procps \
+    python3 \
+    unzip \
+    wget \
+    build-essential \
+    golang-go \
```

Comment:
I have a suggestion about Go in Debian. Please wait for my PR with vulnerability issues fixes